api/src/test/java/marquez/service/models/ModelGenerator.java

```diff
@@ -0,0 +1,157 @@
+package marquez.service.models;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static java.time.temporal.ChronoUnit.HOURS;
+import static marquez.common.models.ModelGenerator.newContext;
+import static marquez.common.models.ModelGenerator.newDatasetId;
+import static marquez.common.models.ModelGenerator.newDatasetIdsWith;
+import static marquez.common.models.ModelGenerator.newDatasetName;
+import static marquez.common.models.ModelGenerator.newDescription;
+import static marquez.common.models.ModelGenerator.newFields;
+import static marquez.common.models.ModelGenerator.newJobType;
+import static marquez.common.models.ModelGenerator.newLocation;
+import static marquez.common.models.ModelGenerator.newNamespaceName;
+import static marquez.common.models.ModelGenerator.newOwnerName;
+import static marquez.common.models.ModelGenerator.newSourceName;
+import static marquez.common.models.ModelGenerator.newTagName;
+import static marquez.common.models.ModelGenerator.newTagNames;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.time.Instant;
+import java.util.stream.IntStream;
+import marquez.Generator;
+import marquez.common.models.DatasetId;
+import marquez.common.models.DatasetName;
+import marquez.common.models.NamespaceName;
+
+/**
+ * Generates new instances for {@code marquez.service.models} with random values used for testing.
+ */
+public final class ModelGenerator extends Generator {
```

Comment:
Can we name these model generators distinctly? E.g., `DbModelGenerator`, `ServiceModelGenerator`, etc.?